### PR TITLE
fix(w3c/headers): noRecTrack text not showing up

### PR DESCRIPTION
**Pull request by marcoscaceres**
Moved some of the stuff around in src/w3c/templates/sotd.js because otherwise it's quite difficult to debug. 

We should eventually move all of them logic into smaller render functions, otherwise the template is too hard to use or maintain.

### CODE REVIEW

**Comment by saschanaz** [comment]
Maybe a test for this?

**Comment by marcoscaceres** [comment]
> Looks 👍 but seems we need a test for this.

Will add one.

File: tests/spec/w3c/headers-spec.js [hunk 1]
```diff
@@ -1283,6 +1283,22 @@ describe("W3C — Headers", () => {
...
+      status: "ED",
+    };
+    Object.assign(ops.config, newProps);
+    const doc = await makeRSDoc(ops);
+    const sotdText = doc
+      .getElementById("sotd")
+      .textContent.replace(/\s+/gm, " ");
+    const expectedString =
+      "does not expect this document to become a W3C Recommendation";
+    expect(sotdText).toMatch(expectedString);
```
Comment (saschanaz) [inline tests/spec/w3c/headers-spec.js]: `toContain()`?

Comment (marcoscaceres) [reply tests/spec/w3c/headers-spec.js]: yeah... copy/🍝:)

**Pull request merged**